filament/include/filament/RenderableManager.h

```diff
@@ -239,9 +241,40 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
          */
         Builder& screenSpaceContactShadows(bool enable) noexcept;
 
+        /**
+         * Allows bones to be swapped out and shared using SkinninBuffer.
+         *
+         * If skinning buffer mode is enabled, clients must call setSkinningBuffer rather than
+         * setBones. This allows sharing of data between renderables.
+         *
+         * @param enabled If true, enables buffer object mode.  False by default.
+         */
+        Builder& enableSkinningBuffers(bool enabled = true) noexcept;
+
+        /**
+         * Enables GPU vertex skinning for up to 255 bones, 0 by default.
+         *
+         * Skinning Buffer mode must be enabled.
+         *
+         * Each vertex can be affected by up to 4 bones simultaneously. The attached
+         * VertexBuffer must provide data in the \c BONE_INDICES slot (uvec4) and the
+         * \c BONE_WEIGHTS slot (float4).
+         *
+         * See also RenderableManager::setSkinningBuffer(), which can be called on a per-frame basis
+         * to advance the animation.
```

Comment:
Nit with this comment.  It would be unusual to advance an animation by swapping out the skinning buffer.  It would be more common for clients to update the bone matrices to their interpolated values by calling `SkinningBuffer::setBones()` on a per-frame basis.